internal/ingress/status/status.go

```diff
@@ -179,19 +180,22 @@ func NewStatusSyncer(config Config) Sync {
 		electionID = fmt.Sprintf("%v-%v", config.ElectionID, config.IngressClass)
 	}
 
+	stopCh := make(chan struct{})
```

Comment:
Why do you need to initialize this variable here? Just declaring would be enough, no?

Reply:
Fixed